pages/basic-features/social-login.mdx

```diff
@@ -0,0 +1,72 @@
+import { Callout, Tabs, Tab } from "nextra-theme-docs";
+
+# Social Login
+
+This feature empowers you to seamlessly integrate social login functionality into your application, enhancing user experience and engagement. Craftable PRO supports various popular social platforms, including Facebook, GitHub, Twitter, Apple, Google, and Microsoft, allowing your users to sign in using their preferred social accounts effortlessly.
+The functionality is powered by the [Laravel Socialite](https://laravel.com/docs/socialite) package and its extensive range of providers.
+
+## Getting Started
+
+To enable social login in your application, follow these simple steps:
+
+1. Select which services you want to use and enable them in `config/craftable-pro.php`:
```

Comment:
instead of numbered list, lets wrap it in `<div className="steps-container">` and make each step h3 (###). Take a look at the `installation.md` to see how it is done there.